core/src/main/java/org/elasticsearch/search/aggregations/pipeline/InternalSimpleValue.java

```diff
@@ -85,4 +85,16 @@ public XContentBuilder doXContentBody(XContentBuilder builder, Params params) th
         }
         return builder;
     }
+
+    @Override
+    protected int doHashCode() {
+        return Objects.hash(value, format.getWriteableName());
```

Comment:
I don't think you need to check `format` here as this seems to be covered by `InternalNumericMetricsAggregation.hashCode()`